config/dev.yaml

```diff
@@ -11,189 +11,246 @@ common:
     # Type: str
     database: "sqlite:///grouper.sqlite"
 
-    # If this exists, it should be the path to an executable that Grouper will run. This program
-    # should print a single SqlAlchemy URL and exit 0.
+    # If this exists, it should be the path to an executable that Grouper will
+    # run. This program should print a single SqlAlchemy URL and exit 0.
     #
     # Type: str
     database_source: ""
 
+    # The format in which to display dates in the interface. More details here:
+    # https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior
+    #
+    # Type: str
+    date_format: "%Y-%m-%d %I:%M %p"
+
+    # Number of days before membership expires from a group at which
+    # notification of the upcoming expiration will be sent.
+    #
+    # Type: int
+    expiration_notice_days: 7
+
     # Format for logging output.
-    # See https://docs.python.org/2/library/logging.html#logrecord-attributes
+    # See https://docs.python.org/3/library/logging.html#logrecord-attributes
+    #
     # Type: str
     log_format: "%(asctime)-15s\t%(levelname)s\t%(message)s  [%(name)s]"
 
-    # Directories for plugins. If set, load plugins from these directories. Plugins can affect the
-    # behavior of Grouper.
-    # Type: list of str
+    # Number of days for approvers of audited groups who are not auditors to
+    # become auditors before being expired out of the group
+    #
+    # Type: int
+    nonauditor_expiration_days: 5
```

Comment:
I believe this is no longer used (since the changes to auto-promote non-auditor approvers) so might as well remove. A quick code search turned up nothing as well.

Reply:
Oh, good call. I'll delete this setting.